src/server/routes/index.ts

```diff
@@ -0,0 +1,37 @@
+import cors from 'cors'
+import CryptoJS from 'crypto-js'
+import { Router } from 'express'
+import { PG_CONNECTION, CRYPTO_KEY } from '../constants'
+import logger from '../logger'
+import oldRoutes from '../../api'
+
+/**
+ * Adds a "pg" object to the request if it doesn't exist
+ */
+const addConnectionToRequest = async (req: any, res: any, next: any) => {
```

Comment:
@soedirgo isn't this the same `addConnectionToRequest` as the one being exported from `src/lib/connectionMiddleware.ts`?

Reply:
@soedirgo 

> minus connectionMiddleware.ts

Oh, I see!